docs/build/tutorials/avalanchego-node/README.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 1
+---
+
+# Overview
+
+This section provides documents on how to build and maintain an AvalancheGo node, and then validate the Avalanche network using an AvalancheGo node.
+
+## Build
+|  |  |
+| :--- | :--- |
+| [**Run an Avalanche Node Using Install Script**](./build/set-up-node-with-installer.md) | How to install and run AvalancheGo |
+| [**Run an Avalanche Node Manually**](./build/run-avalanche-node-manually.md) | How to install without the installer script |
+| [**Set Up an Avalanche Node with Amazon Web Services (AWS)**](./build/cloud/setting-up-an-avalanche-node-with-amazon-web-services-aws.md) | Create a node that runs on AWS |
+| [**Set Up an Avalanche Node with Microsoft Azure**](./build/cloud/set-up-an-avalanche-node-with-microsoft-azure.md) | Create a node that runs on Microsoft Azure |
+| [**Set Up an Avalanche Node with Google GCP**](./build/cloud/set-up-an-avalanche-node-with-google-cloud-platform.md) | Create a node that runs on Google GCP |
+
+
+## Maintain
+|  |  |
+| :--- | :--- |
+| [**Set up Node Monitoring**](./maintain/setting-up-node-monitoring.md) | Set up infrastructure to monitor an instance of AvalancheGo |
+| [**Node Backup and Restore**](./maintain/node-backup-and-restore.md) | Back up important files to be able to restore your node |
+| [**Upgrade Your AvalancheGo Node**](./maintain/upgrade-your-avalanchego-node.mdx) | Upgrade your Avalanche node |
+| [**Run C-Chain Offline Pruning**](./maintain/run-offline-pruning.md) | C-Chain offline pruning |
+| [**Hetzner Volume Migration**](./maintain/hetzner-volume-migration.md) | Hetzner Volume Migration |
```

Comment:
Can we remove this? I don't think it makes sense to have highly specific documentation for this case anymore.

Reply:
@jpop32 what is your opinion on this?

Reply:
Let me see where I can put it